### PR TITLE
Update Naukri.xml and naukimg.com.xml

### PR DESCRIPTION
**Pull request by cschanaj**

### CODE REVIEW

**Comment by J0WI** [comment]
This ruleset should be linked with `Naukri.xml`.

**Comment by cschanaj** [comment]
@J0WI done. thanks!

File: src/chrome/content/rules/Naukri.xml [hunk 1]
```diff
@@ -1,124 +1,38 @@
...
 	<target host="lg.naukri.com" />
+		<test url="http://lg.naukri.com/nLogger/boomLogger.php" />
 	<target host="login.naukri.com" />
 	<target host="naukrirecruiter.naukri.com" />
-	<target host="recruit.naukri.com" />
 	<target host="login.recruit.naukri.com" />
+	<target host="recruit.naukri.com" />
+	<target host="recruiterzone.naukri.com" />
 	<target host="resume.naukri.com" />
-	<!--target host="static.naukri.com" /-->
```
Comment (J0WI) [inline src/chrome/content/rules/Naukri.xml]: This host disappeared silently...

File: src/chrome/content/rules/Naukri.xml [hunk 2]
```diff
@@ -1,124 +1,38 @@
...
-	ᵃ Shows another domain
-	ʳ Refused
-
-
-	Problematic hosts in *naukri.com:
-
-		- ^ ᵈ
-		- careers ᵐ ˢ
-		- static ˣ
-		- w5 ᵐ ˢ
```
Comment (J0WI) [inline src/chrome/content/rules/Naukri.xml]: `w\d` hosts do still exist.

**Pull request merged**